### DISCUSSION TITLE
Show the language KCL on github

### DISCUSSION THREAD

**Discussion by liangyuanpeng** [kcl-lang/kcl#1511] [Ideas]
create a issue to request add a language for show the language tab on github, and I think the KCL official staff may be the right person to submit the issue.

cc @Peefy 

- https://github.com/github-linguist/linguist/issues/new/choose
- https://github.com/github-linguist/linguist

**Comment by Peefy** [comment]
Yes, that's a great idea, but there are strict conditions for highlighting on GitHub: https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#adding-an-extension-to-a-language
In short, we will continue to work hard and highlighting on GitHub is one of the important milestones for the community.

**Reply by Kryan90** [reply]
Would you be able to provide any update on the status of getting kcl added to linguist? Outside of getting some syntax highlighting, it would be hugely beneficial to make it easier to find example KCL code.
